src/components/z-card/readme.md

```diff
@@ -0,0 +1,73 @@
+# z-card
+
+[Abstract](https://app.abstract.com/projects/fd370780-e659-11e8-99dc-0d08537c5fde/branches/e696391e-bd14-4345-a98c-78343d192340/commits/5078c674ea888e961e09ac28741f206cb8506111/files/9f586aa0-bd5b-4c12-9541-aed2e98a7a9c/layers/61FBCDE5-1C49-442B-A262-EA3E73FA352F?collectionId=12a021f0-42e6-4e22-b64a-df001c3d6917&collectionLayerId=9986ef95-6efb-4e31-ae1f-c037d0f16b9d&mode=build&selected=root-45C828A6-52D2-4722-8891-AEEAA2C6E021)
+
+## Implementation state
+
+Variants:
+- [x] Default
+- [x] Border
+- [x] Shadow
+- [x] Overlay
+- [ ] Text
+
+Orientation:
+- [x] Vertical
+- [ ] Horizontal
+
+Interactions:
+- [x] Click
+- [ ] Multi select
+- [ ] Hover actions
+- [ ] Reveal
+
+Cover variants:
+- [x] Horizontal
+- [ ] Vertical
+- [x] With icon
+- [ ] Square
+- [ ] Circle
+
+---
+## Notes
+The width of the card must be set externally, respecting the grid indication of the design.
+
+The CSS variable `--aspect-ratio` can be used to set the ratio of the cover (respecting the indications of the design). Default is `1.62`.
```

Comment:
Describe  the use of  the CSS variable --z-card--color-cover-background to define the cover container background color